test/controllers/admin/comfy/api_namespaces_controller_test.rb

```diff
@@ -467,4 +467,23 @@ class Comfy::Admin::ApiNamespacesControllerTest < ActionDispatch::IntegrationTes
     assert_match success_message, request.flash[:notice]
     assert_equal api_namespace_hash['api_namespace']['name'], ApiNamespace.last.name
   end
+
+  test "should allow #show and the dynamic properties should be shown in the api-resources tables" do
+    sign_in(@user)
+    properties = {"attr_1"=>true, "attr_2"=>true, "attr_3"=>true, "attr_4"=>true}
+    @api_namespace.update(properties: properties)
+    @api_namespace.api_form.update(properties: {
```

Comment:
is `@api_namespace.api_form.update(properties:` updating the API form necessary @alis-khadka ?

Reply:
Just realized that there is a after_save callback for updating the api_form. I have refactored the test case.